.github/CODEOWNERS

```diff
@@ -10,17 +10,17 @@
 /charts/kube-prometheus-stack/ @vsliouniaev @bismarck @gianrubio @gkarthiks @scottrigby @Xtigyro
 /charts/prometheus/ @gianrubio @zanhsieh @Xtigyro @monotek @naseemkullah
 /charts/prometheus-adapter/ @mattiasgees @steven-sheehy @hectorj2f
-/charts/prometheus-blackbox-exporter/ @desaintmartin @gianrubio @monotek @rsotnychenko
+/charts/prometheus-blackbox-exporter/ @desaintmartin @gianrubio @rsotnychenko @monotek
```

Comment:
One thing I just noticed, the check CODEOWNERS Script should sort maintainer names. Now it's probably sorting based on the order in Chart.yaml, but for readability let's throw a sort in that script. And update this PR too. Do you agree?

Reply:
I had it sorted first and was not sure if we want this or want to keep same order as in `Chart.yaml`.
As you raised the issue now as well let's do the sorting ;-)